src/main/java/com/yahoo/bullet/rest/controller/HTTPQueryController.java

```diff
@@ -16,25 +19,36 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
 
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 
 @RestController
 public class HTTPQueryController {
+    private static final String WINDOW_KEY_STRING = "window";
     @Autowired @Setter
     private QueryService queryService;
+    private static final Gson GSON = new GsonBuilder().create();
 
     /**
      * The method that handles POSTs to this endpoint. Consumes the HTTP request, invokes {@link QueryService} to
      * register and transmit the query to Bullet.
      *
-     * @param query The JSON query.
+     * @param queryString The JSON query.
      * @return A {@link CompletableFuture} representing the eventual result.
      */
     @PostMapping(path = "${bullet.endpoint.http}", consumes = { MediaType.TEXT_PLAIN_VALUE }, produces = { MediaType.APPLICATION_JSON_VALUE })
```

Comment:
Can we consume JSON here? Same for SSEMapping below?

Reply:
Yes. Remove them and seems working.

Reply:
I changed it back. Since it will throw json parse exception if the request sending by curl without Content-Type header.

Reply:
Ok, then can we make sure SSEQuery accepts the same thing to make it consistent?

Reply:
Done.